index.html

```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
```

Comment:
We can't use Google's copy of JQuery under SDL guidelines.  See https://liquid.microsoft.com/Web/Object/Read/MS.Security/Requirements/Microsoft.Security.SystemsADM.10103

Reply:
We are already using their copy on the other pages; I just copied off of them. I'll look at replacing all the references.

Reply:
:'(